File: Python Programming/problems on numbericals/problem91.py
'''
91. Display ASCII value of given character
	
	Author :Milind Nalawade
	
	Date : 13/07/2021
	
	Language : Java 14.0.2

'''

#====
#Logic
#====
def DisplayASCII(ch):
	print("\n The ASCII value of ",ch," is = ",ord(ch))
	
#=============
#Entry point function
#=============
def main():
	ch=input("\n Enter your character : ")
	DisplayASCII(ch)
	
#=====
#starter
#=====
if __name__=="__main__":
	main()

File: C Programming/problem on numbericals/problem91.c
/*
91. Display ASCII value of given character
	
	Author :Milind Nalawade
	
	Date : 13/07/2021
	
	Language : C programming
	
*/

#include<stdio.h>

void DisplayASCII(char ch)
{
	printf("\n The ASCII value of the %c is = %d \n",ch,ch);
}

int main()
{
	char ch='\0';
	printf("\n Enter your character : ");
	scanf("%c",&ch);
	DisplayASCII(ch);
	return 0;
}	
